docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -25,308 +29,438 @@ function su_humsci_profile_final_task() {
 }
 
 /**
- * Indicates version of profile.
- */
-function su_humsci_profile_update_8200() {
-  // Intentionally left empty.
-}
-
-/**
- * Set the permissions for the hero overlay field.
+ * Implements hook_update_last_removed().
  */
-function su_humsci_profile_update_8201() {
-  $perms = [
-    'view field_hs_hero_overlay_color',
-    'view own field_hs_hero_overlay_color',
-  ];
-  user_role_grant_permissions(UserInterface::ANONYMOUS_ROLE, $perms);
-  $perms = [
-    'create field_hs_hero_overlay_color',
-    'edit field_hs_hero_overlay_color',
-    'edit own field_hs_hero_overlay_color',
-    'view field_hs_hero_overlay_color',
-    'view own field_hs_hero_overlay_color',
-  ];
-  user_role_grant_permissions(UserInterface::AUTHENTICATED_ROLE, $perms);
+function su_humsci_profile_update_last_removed() {
```

Comment:
What's this do?

Reply:
https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_update_last_removed/10

When deleting update hooks you have to add/update this to the last update hook that was deleted.